recipes/freecad/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "freecad" %}
+{% set version = "0.18b0" %}
+{% set sha256 = "1462df34caa67791bb7512f6c464f16a15d49f0d9f777996cb4e7b8174c57191" %}
+{% set commit = "e49a5af3ae2f976017f1156bdc113594f81d3fb5" %}
+
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}_{{ version }}.tar.gz
+    url: https://github.com/FreeCAD/FreeCAD/archive/{{ commit }}.tar.gz
+    sha256: {{ sha256 }}
+    patches:
+        - osx.patch
+        - smesh.patch
+        - win_unicode.patch
+
+build:
+    number: 0
+    skip: True  # [py < 36]
+
+requirements:
+    build:
+        - {{ compiler("cxx") }}
+        - vc 14  # [win]
+        - ninja
+        - cmake
+    host:
+        - nomkl  # [unix]
+        - boost-cpp 1.66.0
+        - python
+        - qt 5.6.*
+        - occt 7.2.*
+        - xerces-c 3.2.*
+        - zlib 1.2.*
+        - vtk 8.1.1
+        - swig
+        - eigen
+        - pybind11
+        - matplotlib
+        - coin3d
+        - pivy
+        - pyside2
+        - smesh
+        - netgen  # [not win32]
+        - doxygen
+        - libglu  # [linux]
+        - mesalib  # [linux]
+        - gcc  # [linux]
```

Comment:
Remove this

Reply:
Can you send me a log about the linking problems?

Reply:
https://circleci.com/gh/conda-forge/staged-recipes/27570
I guess it was this issue. But I am not that sure.
```
CMake Error at /home/conda/feedstock_root/build_artifacts/freecad_1527654236437/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/cmake/Qt5Gui/Qt5GuiConfigExtras.cmake:9 (message):
  Failed to find "GL/gl.h" in
```

Reply:
I tried to reproduce locally, and this are the errors I was referring to:
```
/opt/rh/devtoolset-2/root/usr/libexec/gcc/x86_64-redhat-linux/4.8.2/ld: cannot find -lpython3.6m
/opt/rh/devtoolset-2/root/usr/libexec/gcc/x86_64-redhat-linux/4.8.2/ld: cannot find -lxerces-c
/opt/rh/devtoolset-2/root/usr/libexec/gcc/x86_64-redhat-linux/4.8.2/ld: cannot find -lboost_filesystem
/opt/rh/devtoolset-2/root/usr/libexec/gcc/x86_64-redhat-linux/4.8.2/ld: cannot find -lboost_program_options
/opt/rh/devtoolset-2/root/usr/libexec/gcc/x86_64-redhat-linux/4.8.2/ld: cannot find -lboost_regex
/opt/rh/devtoolset-2/root/usr/libexec/gcc/x86_64-redhat-linux/4.8.2/ld: cannot find -lboost_signals
/opt/rh/devtoolset-2/root/usr/libexec/gcc/x86_64-redhat-linux/4.8.2/ld: cannot find -lboost_system
/opt/rh/devtoolset-2/root/usr/libexec/gcc/x86_64-redhat-linux/4.8.2/ld: cannot find -lboost_thread
/opt/rh/devtoolset-2/root/usr/libexec/gcc/x86_64-redhat-linux/4.8.2/ld: cannot find -lboost_chrono
/opt/rh/devtoolset-2/root/usr/libexec/gcc/x86_64-redhat-linux/4.8.2/ld: cannot find -lboost_date_time
/opt/rh/devtoolset-2/root/usr/libexec/gcc/x86_64-redhat-linux/4.8.2/ld: cannot find -lboost_atomic
```

Reply:
@isuruf  any idea why the linking behaviour is different?

Reply:
I need a full log. Do you have a CircleCI log with the linking issues?

Reply:
here is one with the same proiblem:
https://circleci.com/gh/conda-forge/staged-recipes/27451

Reply:
It seems to be missing things like `-L${PREFIX}/lib` and `-I${PREFIX}/include`. Not sure why that is. 😕